docs/topics/advent-of-code.md

```diff
@@ -0,0 +1,80 @@
+[//]: # (title: Advent of Code puzzles in idiomatic Kotlin)
+
+[Advent of Code](https://adventofcode.com/) is an annual series of Christmas-themed puzzles that follow an Advent calendar,
+where a new puzzle is released each day. With the permission of [Eric Wastl](http://was.tl/), creator of Advent of Code,
+we’ll show how to solve these puzzles using the idiomatic Kotlin style.
+
+> You can find all the solutions for the Advent of Code 2020 puzzles in our [GitHub repository](https://github.com/kotlin-hands-on/advent-of-code-2020/).
+>
+
+## Day 1: Report repair
+
+In this puzzle, you'll explore input handling, iterating over a list, different ways of building a map, and using
```

Comment:
To check: I think a lot of our materials are using "we" and "we'll" - just like your first paragraph. It might make sense to continue this here in the main body.